common/defs-cmds.go

```diff
@@ -81,243 +81,328 @@ const (
+Use the command options below to change the information returned by this command.`
 
-const WorkflowUsageText = `Workflow commands allow operations to be performed on [Workflow Executions](/concepts/what-is-a-workflow-execution).
+const WorkflowUsageText = `[Workflow](/concepts/what-is-a-workflow) commands perform operations on on [Workflow Executions](/concepts/what-is-a-workflow-execution).
+
+Workflow commands use this syntax:
+` + "`" + `temporal workflow COMMAND [ARGS]` + "`" + `
 `
 
 const StartWorkflowUsageText = `The ` + "`" + `temporal workflow start` + "`" + ` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution).
-When invoked successfully, the Workflow and Run ID are returned immediately after starting the [Workflow](/concepts/what-is-a-workflow).
+The Workflow and Run IDs are returned after starting the [Workflow](/concepts/what-is-a-workflow).
+
+` + "`" + `temporal workflow start --task-queue=MyTaskQueue --type=MyWorkflow` + "`" + `
 
-Use the command options listed below to change how the Workflow Execution behaves upon starting.
-Make sure to write the command in this format:
-` + "`" + `temporal workflow start [command options]` + "`"
+Use the command options below to change the information returned by this command.`
 
 const ExecuteWorkflowUsageText = `The ` + "`" + `temporal workflow execute` + "`" + ` command starts a new [Workflow Execution](/concepts/what-is-a-workflow-execution) and prints its progress.
-The command doesn't finish until the [Workflow](/concepts/what-is-a-workflow) completes.
+The command completes when the Workflow Execution completes.
+
+[Workflows](/concepts/what-is-a-workflow) are executed with the following syntax:
+` + "`" + `temporal workflow execute --workflow-id=meaningful-business-id --type=MyWorkflow --task-queue=MyTaskQueue` + "`" + `
 
 Single quotes('') are used to wrap input as JSON.
 
-Use the command options listed below to change how the Workflow Execution behaves during its run.
-Make sure to write the command in this format:
-` + "`" + `temporal workflow execute [command options]` + "`"
+` + "`" + `temporal workflow execute --workflow-id=meaningful-business-id --type-MyWorkflow --task-queue-MyTaskQueue --input='{"JSON": "Input"}'` + "`" + `
+
+Use the command options below to change the information returned by this command.`
 
 const DescribeWorkflowUsageText = `The ` + "`" + `temporal workflow describe` + "`" + ` command shows information about a given [Workflow Execution](/concepts/what-is-a-workflow-execution).
+
 This information can be used to locate Workflow Executions that weren't able to run successfully.
 
-Use the command options listed below to change the information returned by this command.
-Make sure to write the command in this format:
-` + "`" + `temporal workflow describe [command options]` + "`"
+` + "`" + `temporal workflow describe --workflow-id=meaningful-business-id` + "`" + `
```

Comment:
does this look good when printed with --help from the binary? Making sure the extra lines do not mess up --help output

Reply:
yeah, it looked fine